books/upload_to_snowflake.py

```diff
@@ -0,0 +1,49 @@
+from snowflake.snowpark.session import Session
+
+if __name__ == '__main__':
+    snowflake = Session.builder.configs({
+        "connection_name" : "dev"
+    }).create()
+    snowflake.sql("""
+        CREATE schema if not exists raw.aleks
+        """).show()
+    snowflake.sql("""
+    CREATE OR REPLACE STAGE raw.aleks.books_stage
+    """).show()
+    snowflake.sql("""
+        CREATE OR REPLACE TEMP TABLE raw.aleks.tmp_books(
+            Title varchar,
+            Author varchar,
+            ISBN_ISSN varchar,
+            DOI varchar,
+            Wydawnictwo varchar,
+            Rok_wydania varchar,
+            Liczba_stron varchar,
+            XML_ISBN_ISSN varchar
```

Comment:
clean up the columns names (those are not case sensitive but keeping them unified enhances the readability)